### ISSUE TITLE
Question about validation/test set

### ISSUE THREAD

**Issue by DeepRunner** [xingyizhou/pytorch-pose-hg-3d#6]
The video is down-sampled from 50 fps to 0.25 fps in the validation set.
Please see [GetH36M.py](https://github.com/xingyizhou/pytorch-pose-hg-3d/blob/master/src/tools/GetH36M.py#L50). 
Is this validation set also the test set to produce the Table 1 in your paper?

Thank you very much!

**Comment by xingyizhou** [comment]
Hi, 
  Thanks for your interests in this project. No, it is only a sampled test set for fast evaluation. For the full evaluation, you need to uncomment the two lines in GetH36M.py. The finally number of the pytorch code might be slightly different from the paper. Please use to lua torch code to get the best performance (which is actually better than the reported number in paper).

**Comment by DeepRunner** [comment]
Thanks for the explanation.
